mpack-instance-manager/src/main/python/instance_manager/mpack-instance-manager.py

```diff
@@ -152,24 +155,44 @@ def main(options, args):
                        components_map=parsed_components_map)
 
   elif action == GET_CONF_DIR_ACTION:
-    print get_conf_dir(mpack=options.mpack, mpack_instance=options.mpack_instance,
+    if options.format:
+      print json.dumps(get_conf_dir(mpack=options.mpack, mpack_instance=options.mpack_instance,
                        subgroup_name=options.subgroup_name, module_name=options.module_name,
-                       components_map=parsed_components_map)
+                       components_map=parsed_components_map), indent=INDENT)
+    else:
+      print json.dumps(get_conf_dir(mpack=options.mpack, mpack_instance=options.mpack_instance,
+                                    subgroup_name=options.subgroup_name, module_name=options.module_name,
+                                    components_map=parsed_components_map))
```

Comment:
Can you please reduce code duplication by always passing the appropriate `indent`?

```
indent = INDENT if options.format else None
print json.dumps(get_conf_dir(...), indent=indent)
```

(Also in other similar places.)

Reply:
Will update